bip-0068.mediawiki

```diff
@@ -49,6 +49,8 @@ The block produced time is equal to the median-time-past of its previous block.
 
 When the relative lock-time is block-based, it is interpreted as a minimum block-height constraint over the input's age. A relative block-based lock-time of zero indicates an input which can be included in any block. More generally, a relative block lock-time n can be included n blocks after the mining date of the output it is spending, or any block thereafter.
 
+The new rules are not applied to the nSequence field of the input of generation transaction.
```

Comment:
"of the generation"

Is this the proper terminology? Should we say "coinbase transaction" here? Honestly not sure what the current accepted terminology is.